kernel/arch/dreamcast/include/dc/syscalls.h

```diff
@@ -0,0 +1,395 @@
+/* KallistiOS ##version##
+
+   dc/syscalls.h
+   Copyright (C) 2024 Andy Barajas
+*/
+
+/** \file      dc/syscalls.h
+    \brief     Functions to access the system calls of the Dreamcast ROM.
+    \ingroup   system_calls
+
+    \author Marcus Comstedt
+    \author Andy Barajas
+*/
+
+/** \defgroup  system_calls System Calls
+    \brief     API for the Dreamcast's system calls
+    \ingroup   system
+
+    This module encapsulates all the system calls available in the Dreamcast 
+    Operating System, allowing direct interaction with system hardware 
+    components such as the GDROM drive, flash ROM, and bios fonts. These 
+    functions are essential for performing low-level operations that are not
+    handled by standard user-space APIs.
```

Comment:
```suggestion
    handled by standard user-space APIs.
    
    @{
```
Do this and you can remove the `\ingroup system_calls` command from every single public API method and clean it up. This will include them all automatically in the enclosing group.